File: js/packages/native-modules/PushTokenRequester/PushTokenRequester.ts
import { PushTokenRequesterInterface } from './PushTokenRequesterInterface'

class NoopPushTokenRequester implements PushTokenRequesterInterface {
	request(): Promise<string> {
		return Promise.reject()
	}
}

export const PushTokenRequester: PushTokenRequesterInterface = new NoopPushTokenRequester()

File: berty-bridge-expo/ios/Sources/PushTokenRequester/PushTokenRequester.m
//
//  PushTokenRequester.m
//  Berty
//
//  Created by Antoine Eddi on 01/09/2021.
//

#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(PushTokenRequester, NSObject)

RCT_EXTERN_METHOD(request:(RCTPromiseResolveBlock)resolve
                  reject:(RCTPromiseRejectBlock)reject);

@end
